### PR TITLE
✨ uikit distribution build and publish

### PR DESCRIPTION
**Pull request by hlminh2000**
- uikit distribution build and npm publish setup.
- also adds a test build of `uikit` to CI test stage

Approach: 
**Build steps:**
**1) edit `version` under `uikit/package.release.json`**
**2) run `npm run build-uikit` which performs the following:**
  - removes `uikit/dist` folder (the uikit build result, gitignored)
  - creates a TypeScript build, outputting to `uikit/dist`, outputting `.jsx` files
  - transpiles `.jsx` files from step 2 with Babel to resolve absolute imports, jsx and emotion `css` props. outputting `.js` files.
  - copies static assets from `/uikit` to `uikit/dist`, including:
    + `uikit/assets` -> `uikit/dist/assets` directory containing images, etc...
    + `uikit/readme.md` (this gets presented at https://www.npmjs.com/package/@icgc-argo/uikit)
    + `uikit/package.release.json` -> `uikit/dist/package.json` (npm will use this as package definition for publish)
  - cleanup: removes `.jsx` by-products from `uikit/dist`

**Publish:** `npm run publish-uikit`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -73,4 +73,5 @@ chromedriver.log
 .vscode
 
 .graphqlconfig
-.cache
\ No newline at end of file
+.cache
+uikit/dist
```
Comment (joneubank) [inline .gitignore]: 👍

File: Jenkinsfile.groovy [hunk 2]
```diff
@@ -46,6 +46,7 @@ spec:
                     sh "npm run type-check"
                     sh "npm run build"
                     sh "npm run test"
+                    sh "npm run build-uikit"
```
Comment (joneubank) [inline Jenkinsfile.groovy]: Why is this needed if we arent publishing the UI automagically?

Comment (hlminh2000) [reply Jenkinsfile.groovy]: just a test build to ensure it builds :P
